### PR TITLE
Add Amazon Linux 2

### PR DESCRIPTION
**Pull request by mikemadden42**

### CODE REVIEW

**Comment by mikemadden42** [comment]
Tests are passing for this Amazon Linux 2 box:

```
PLAY RECAP **************************************************************************************************************************************************************
tester-amzn2               : ok=319  changed=145  unreachable=0    failed=0
```

**Comment by andrewkroh** [comment]
In case anyone else wonders how this image is made: https://github.com/gebailey/vagrant-boxes/tree/master/amzn2

File: Vagrantfile [hunk 1]
```diff
@@ -130,6 +130,14 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     testvm.vm.provision "python", type: "shell", inline: ubuntu_provision_python()
   end
 
+  config.vm.define "tester-amzn2" do |testvm|
```
Comment (andrewkroh) [inline Vagrantfile]: To be sort of consistent with the other Amazon image I suggest calling it tester-awslinux2.

Comment (mikemadden42) [reply Vagrantfile]: I renamed the box to awslinux2.

**Pull request merged**